stubs/cefpython3/cefpython3/cefpython.pyi

```diff
@@ -0,0 +1,741 @@
+ERR_CONNECTION_CLOSED: int = ...
+ERR_CONNECTION_FAILED: int = ...
+ERR_CONNECTION_REFUSED: int = ...
+ERR_CONNECTION_RESET: int = ...
+ERR_DISALLOWED_URL_SCHEME: int = ...
+ERR_EMPTY_RESPONSE: int = ...
+ERR_FAILED: int = ...
+ERR_FILE_NOT_FOUND: int = ...
+ERR_FILE_TOO_BIG: int = ...
+ERR_INSECURE_RESPONSE: int = ...
+ERR_INTERNET_DISCONNECTED: int = ...
+ERR_INVALID_ARGUMENT: int = ...
+ERR_INVALID_CHUNKED_ENCODING: int = ...
+ERR_INVALID_HANDLE: int = ...
+ERR_INVALID_RESPONSE: int = ...
+ERR_INVALID_URL: int = ...
+ERR_METHOD_NOT_SUPPORTED: int = ...
+ERR_NAME_NOT_RESOLVED: int = ...
+ERR_NO_SSL_VERSIONS_ENABLED: int = ...
+ERR_NOT_IMPLEMENTED: int = ...
+ERR_RESPONSE_HEADERS_TOO_BIG: int = ...
+ERR_SSL_CLIENT_AUTH_CERT_NEEDED: int = ...
+ERR_SSL_PROTOCOL_ERROR: int = ...
+ERR_SSL_RENEGOTIATION_REQUESTED: int = ...
+ERR_SSL_VERSION_OR_CIPHER_MISMATCH: int = ...
+ERR_TIMED_OUT: int = ...
+ERR_TOO_MANY_REDIRECTS: int = ...
+ERR_TUNNEL_CONNECTION_FAILED: int = ...
+ERR_UNEXPECTED: int = ...
+ERR_UNEXPECTED_PROXY_AUTH: int = ...
+ERR_UNKNOWN_URL_SCHEME: int = ...
+ERR_UNSAFE_PORT: int = ...
+ERR_UNSAFE_REDIRECT: int = ...
+
+FOCUS_SOURCE_NAVIGATION: int = ...
+FOCUS_SOURCE_SYSTEM: int = ...
+
+JSDIALOGTYPE_ALERT: int = ...
+JSDIALOGTYPE_CONFIRM: int = ...
+JSDIALOGTYPE_PROMPT: int = ...
+
+KeyEventType: TypeAlias = int
+KEYEVENT_RAWKEYDOWN: int = ...
+KEYEVENT_KEYDOWN: int = ...
+KEYEVENT_KEYUP: int = ...
+KEYEVENT_CHAR: int = ...
+
+KeyEventFlags: TypeAlias = int
+EVENTFLAG_NONE: int = ...
+EVENTFLAG_CAPS_LOCK_ON: int = ...
+EVENTFLAG_SHIFT_DOWN: int = ...
+EVENTFLAG_CONTROL_DOWN: int = ...
+EVENTFLAG_ALT_DOWN: int = ...
+EVENTFLAG_LEFT_MOUSE_BUTTON: int = ...
+EVENTFLAG_MIDDLE_MOUSE_BUTTON: int = ...
+EVENTFLAG_RIGHT_MOUSE_BUTTON: int = ...
+if sys.platform == "darwin":
+    EVENTFLAG_COMMAND_DOWN: int = ...
+    EVENTFLAG_NUM_LOCK_ON: int = ...
+    EVENTFLAG_IS_KEY_PAD: int = ...
+    EVENTFLAG_IS_LEFT: int = ...
+    EVENTFLAG_IS_RIGHT: int = ...
+
+LOGSEVERITY_VERBOSE: int = ...
+LOGSEVERITY_INFO: int = ...
+LOGSEVERITY_WARNING: int = ...
+LOGSEVERITY_ERROR: int = ...
+LOGSEVERITY_DISABLE: int = ...
+
+STATE_DEFAULT: int = ...
+STATE_ENABLED: int = ...
+STATE_DISABLED: int = ...
+
+PaintElementType: TypeAlias = int
+PET_VIEW: int = ...
+PET_POPUP: int = ...
+
+TID_UI: int = ...
+TID_FILE: int = ...
+TID_FILE_BACKGROUND: int = ...
+TID_FILE_USER_VISIBLE: int = ...
+TID_FILE_USER_BLOCKING: int = ...
+TID_IO: int = ...
+TID_RENDERER: int = ...
+
+TerminationStatus: TypeAlias = int
+TS_ABNORMAL_TERMINATION: int = ...
+TS_PROCESS_WAS_KILLED: int = ...
+TS_PROCESS_CRASHED: int = ...
+
+WindowOpenDisposition: TypeAlias = int
+WOD_UNKNOWN: int = ...
+WOD_CURRENT_TAB: int = ...
+WOD_SINGLETON_TAB: int = ...
+WOD_NEW_FOREGROUND_TAB: int = ...
+WOD_NEW_BACKGROUND_TAB: int = ...
+WOD_NEW_POPUP: int = ...
+WOD_NEW_WINDOW: int = ...
+WOD_SAVE_TO_DISK: int = ...
+WOD_OFF_THE_RECORD: int = ...
+WOD_IGNORE_ACTION: int = ...
+
+def CreateBrowserSync(
```

Comment:
This looks different from the arguments in https://github.com/cztomczak/cefpython/blob/5679f28cec18a57a56e298da2927aac8d8f83ad6/src/cefpython.pyx#L654